tests/system/python/pair/test_e2e_fledge_pair.py

```diff
@@ -81,7 +81,13 @@ def reset_and_start_fledge_remote(self, storage_plugin, remote_user, remote_ip,
                                                                                                       remote_ip,
                                                                                                       remote_fledge_path)], shell=True, check=True)
         storage_plugin_val = "postgres" if storage_plugin == 'postgres' else "sqlite"
-        subprocess.run(["ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i {} {}@{} echo $(jq -c --arg STORAGE_PLUGIN_VAL {} '.plugin.value=$STORAGE_PLUGIN_VAL' {}/data/etc/storage.json) > {}/data/etc/storage.json".format(key_path, remote_user, remote_ip, storage_plugin_val, remote_fledge_path, remote_fledge_path)], shell=True, check=True)
+        # Check whether storage.json file exist on remote machine or not, if it doesn't exist then raise assertion otherwise update its storage plugin value.
```

Comment:
We should avoid the storage.json existence check as reset fixture should be complete reset.
Before running the test machine should be in clean state.

Reply:
@ashish-jabble I think at this stage of test, instance must have a storage JSON file.

Reply:
Correct. As storage.json will be created on fledge start and even on fledge kill/stop commands.